guides/common/modules/ref_overview-of-individual-loggers.adoc

```diff
@@ -1,53 +1,54 @@
-[id="Enabling_Individual_Loggers_{context}"]
-= Enabling Individual Loggers
-
-You can enable individual loggers for selective logging.
-{Project} uses the following loggers:
+[id="Overview_of_Individual_Loggers_{context}"]
+= Overview of Individual Loggers
+The following list provides an overview of selected loggers. You can find the complete list of loggers with their default values in
```

Comment:
Some thought: I think it's a poor practice to suggest users to read the source code. Perhaps we can create a `foreman-rake logging:list` (or similarly named) task. Or at least some supported way. Perhaps even generate the whole table.

Reply:
Regarding referencing source code: I get what you mean, it's not a pretty solution. But when I weighed the pros and cons, I couldn't really come up with a logical reason not to do it. So I decided to keep the reference.

In general, Red Hat writers are constantly being asked to reduce the amount of content we maintain. This is one way to do that -- to avoid listing options, components, etc. and instead link to their primary source wherever possible.

For the record, I filed https://github.com/theforeman/foreman-documentation/issues/2403 about this.

Reply:
Nothing actionable here right now, as far as I can see (there is an agreement on a short-term solution in this thread). Resolving. Feel free to correct me if I got it wrong.